test/runtime/configMatters/launch/jhh/reservation/reservation.py

```diff
@@ -27,14 +34,19 @@ def runCmd(cmd, env=None):
 
 def skipif():
     global skipReason
-    # These tests only run on slurm-srun
-    output = runCmd("printchplenv --simple")
-    for line in output.splitlines():
-        (key, value) = line.split('=',1)
-        if key == 'CHPL_LAUNCHER' and value != 'slurm-srun':
-            skipReason = "CHPL_LAUNCHER != slurm-srun"
-    # Verify environment variables
-    if os.environ.get('CHPL_RT_LOCALES_PER_NODE', None) != '1':
+
+    # Get the Chapel configuration
+    printchplenv.compute_all_values()
+    # strip the padding printchplenv puts on some of the keys
+    env = {k.strip():v for k,v in printchplenv.ENV_VALS.items()}
+
+    # Verify Chapel configuration
+    if env.get('CHPL_LAUNCHER', None) != 'slurm-srun':
+        skipReason = "CHPL_LAUNCHER != slurm-srun"
+        return
```

Comment:
Should `return` be added to the other checks as well?

Reply:
Done.